### PR TITLE
Update StartHere.md

### PR DESCRIPTION
**Pull request by kouts**
Update text in order to reflect latest changes in topic selection.

### CODE REVIEW

File: StartHere.md [hunk 1]
```diff
@@ -20,22 +20,17 @@ Discuss issues you've faced, discover new & better ways of working (together), c
...
 <a href="https://www.meetup.com/devstaff/"><img src="images/meetup.png" align="left"  height="80" ></a>
 
-The main tool for organizing our meetings is *meetup.com*. The first step is head [there](https://www.meetup.com/devstaff/) and join the group. 
+The main tool for organizing our meetings is *meetup.com*. The first step is to head [there](https://www.meetup.com/devstaff/) and join the group. 
 
-Make sure you configure your notification options on meetup.com to receive notifications about upcoming events!
+Make sure you configure your notification options on meetup.com in order to receive notifications about upcoming events!
 
+## Join the chat
+A big part of the community revolves around our Slack group where you can find the latest tech news, interesting articles, random pieces of information and a lot of heated discussions! Click on the link [here](https://join.slack.com/t/devstaff/shared_invite/enQtNDYyNTI0NjUyMjczLWUyN2ZkOGJjYWQyMzU2MTE0MjViYTAxYWYxMjVlMzk2ZDk1N2I3ZGI2Y2MxMmMwN2JkNzY5MGUzMGRmN2NlNWM) to join.
```
Comment (DaKnOb) [inline StartHere.md:28]: `s/information and/information, and/`

File: StartHere.md [hunk 2]
```diff
@@ -20,22 +20,17 @@ Discuss issues you've faced, discover new & better ways of working (together), c
...
+The Org team is responsible for curating topics and finding speakers for the upcoming meetups. Any member of the community is free to suggest new topics and also propose himself/herself for being the next meetup speaker on his/her favorite topic, by filling this form [here](https://docs.google.com/forms/d/e/1FAIpQLSd-xubuaYZIl0kl1B5wqy_nIE-Rq010OfUEfAdMyoSA6S9jAg/viewform). The DevStaff Slack channel is also a great place to propose new topics to the Org team as well as pointing to speakers/topics that might be of interest to the community. So if you want to propose a topic, or you are available to make a presentation on a topic, go ahead and fill in the [form](https://docs.google.com/forms/d/e/1FAIpQLSd-xubuaYZIl0kl1B5wqy_nIE-Rq010OfUEfAdMyoSA6S9jAg/viewform) or head out to our Slack channel and drop us a line there! When there's interest in a certain topic the Org team actively tries to find more speakers willing to talk about it. There's always someone sharing the same passion with you!
```
Comment (DaKnOb) [inline StartHere.md:32]: `s/propose himself\/herself/volunteer themselves/`

Comment (DaKnOb) [inline StartHere.md:32]: `s/his\/her/their/`

Comment (DaKnOb) [inline StartHere.md:32]: `s/make a presentation on/present/`

Comment (DaKnOb) [inline StartHere.md:32]: `s/a certain topic/a certain topic,/`

Comment (mzampetakis) [inline StartHere.md:32]: If we have a short URL for the form I would advise using it here as it keeps some statistics.

**Review by DaKnOb — CHANGES_REQUESTED** [review]
Some additional points on top of the comments made:

* We should be consistent with the `Meet Up` word. It has several different spellings / capitalizations, we should pick one, and stick with it, unless we refer to the meetup.com platform which is branded as `Meetup`. I suggest we use this spelling for all of them.
* I'd like to see the document reformatted to 80 characters per line, as most editors can do this automatically, and it makes commenting on the exact phrase easier than the current version where each paragraph is a single line. Of course, this is up for discussion.

File: StartHere.md [hunk 3]
```diff
@@ -20,22 +20,17 @@ Discuss issues you've faced, discover new & better ways of working (together), c
 
 <a href="https://www.meetup.com/devstaff/"><img src="images/meetup.png" align="left"  height="80" ></a>
 
-The main tool for organizing our meetings is *meetup.com*. The first step is head [there](https://www.meetup.com/devstaff/) and join the group. 
+The main tool for organizing our meetings is *meetup.com*. The first step is to head [there](https://www.meetup.com/devstaff/) and join the group. 
 
-Make sure you configure your notification options on meetup.com to receive notifications about upcoming events!
+Make sure you configure your notification options on meetup.com in order to receive notifications about upcoming events!
```
Comment (mzampetakis) [inline StartHere.md:25]: We could mention here that we use meetup notifications for other activities and news of the community apart from the upcoming meetups.

File: StartHere.md (hunk 2 above)
Comment (DaKnOb) [inline StartHere.md:32]: I think we should use `Org Team` instead of `Org team` throughout the document, as this is the full name.

**Review by DaKnOb — CHANGES_REQUESTED** [review]

**Review by mzampetakis — CHANGES_REQUESTED** [review]
Great update here! Just a couple of comments!
Also if possible change the line length to be up to 80 chars for easier reviewing in the future.

**Review by mzampetakis — DISMISSED** [review]

**Review by sonaht — APPROVED** [review]
Looks ok to me !!!

File: StartHere.md (hunk 2 above)
Comment (kouts) [reply StartHere.md]: This is now fixed with 4f6190a

**Review by kouts — COMMENTED** [review]
This is now fixed with 4f6190a

**Review by DaKnOb — DISMISSED** [review]

**Review by DaKnOb — APPROVED** [review]

**Pull request merged**